blog/models.py

```diff
@@ -179,3 +173,25 @@ def _validate_recommended_articles_uniqueness(self) -> None:
                 raise ValidationError(message=f"'{stream_child.value}' is listed more than once!")
             else:
                 article_pages_set.add(stream_child.value)
+
+
+@receiver(post_save, sender=BlogArticlePage)
+def ensure_main_article_exists(sender: Any, instance: Any, **_kwargs: Dict[Any, Any]) -> None:
```

Comment:
Is really `sender` and `instance` need to be "Any"? Won't it be always here instance of `BlogArticlePage`?

Reply:
I suppose so, I guess my outlook here was too abstract. 😃 While we're on the topic here: should kwargs be described with `Dict[Any, Any]` or a more precise `Dict[str, Any]`.